### PR TITLE
Shovel mulch (very imperfect rn)

### LINKED ISSUE #1431: Ability to remove mulch from plots
make it to where there can be multiple mulchs on one plot of the farm

### PR DESCRIPTION
**Pull request by Symi001**
Current issues :
- Display isn't nice, I've opted for a display in the Mulch list but it's not of the right size and kinda weird. Any idea on what's best for the display is more than welcome :)
- I'm still unsure if MulchShovelItem class definition should have its own file, I'd need some input on this :)
- ~I'm really bad with art... so I did not make another shovel art. If anyone can use anything better than Paint, I'm all in for it !~ Thx NiceDice for this point :)

Closes #1431

### CODE REVIEW

**Comment by Symi001** [comment]
Forgot to mention that mulch shovel are counted towards the shovel count, don't hesitate to tell me if you want me to add another variable for it ;)

**Comment by jk13pclick** [comment]
Linked Issue #1431.

**Comment by RedSparr0w** [comment]
Looking good!

May be worth swapping the normal shovel with the mulch shovel when you swap to the mulch tab:

File: src/scripts/farming/Farming.ts [hunk 1]
```diff
@@ -1146,6 +1149,27 @@ class Farming implements Feature {
...
+        if (!plot.isUnlocked) {
+            return;
+        }
+        if (this.mulchShovelAmt() <= 0) {
+            return;
+        }
+
+        if (plot.clearMulch()) {
+            GameHelper.incrementObservable(this.mulchShovelAmt, -1);
+            //GameHelper.incrementObservable(App.game.statistics.totalShovelsUsed, 1);
```
Comment (Symi001) [inline src/scripts/farming/Farming.ts:1167]: I'm still unsure about this, should I increment it ? Should I add another variable to statistics, this time specific to mulch shovel ?

**Comment by Symi001** [comment]
Hello Red, anything else you'd want changed on the review ?

File: src/scripts/farming/Farming.ts (hunk 1 above)
Comment (RedSparr0w) [reply src/scripts/farming/Farming.ts:1167]: Hmm, I think either option works.
A new statistic, or just use the existing shovel statistic.

**Review by RedSparr0w — APPROVED** [review]
Looks great, thanks for the PR! 👍

**Pull request merged**